compiler/rustc_target/src/spec/abi.rs

```diff
@@ -149,7 +149,9 @@ pub fn is_stable(name: &str) -> Result<(), AbiDisabled> {
     match name {
         // Stable
         "Rust" | "C" | "cdecl" | "stdcall" | "fastcall" | "aapcs" | "win64" | "sysv64"
-        | "system" | "efiapi" => Ok(()),
+        | "system" | "efiapi" | "C-unwind" | "cdecl-unwind" | "stdcall-unwind"
+        | "fastcall-unwind" | "aapcs-unwind" | "win64-unwind" | "sysv64-unwind"
+        | "system-unwind" => Ok(()),
```

Comment:
Nit: I wonder if this list would be better off sorted or otherwise grouped… As it is now you first find e.g. `"C"` and then a dozen patterns later a related `"C-unwind"`, they'd be probably better off grouped together.